floodviz/thumbnail/thumbnail.js

```diff
@@ -82,7 +87,7 @@ function convert(figure, window, css_path, filename) {
 		svg_string = inject_style(style_default, null, svg, window);
```

Comment:
This comment is for line 78 where the additional css file is read in. Can this be done after line 30.? This gets the reading of the file done where the config parameter is read. You should be able to assign the value to style_ext and then you only need a single line to inject_style(style_default, style_ext, svg, window).